dev-libs/plasma-wayland-protocols/plasma-wayland-protocols-9999.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=8
 
 ECM_NONGUI=true
-KFMIN=5.82.0
+KFMIN=5.240.0
```

Comment:
Certainly doable, but what does that mean for stable Qt5 revdeps? We can't stabilise further `dev-libs/plasma-wayland-protocols` versions without pulling in Qt6 for them?